dset-requirements.txt

```diff
@@ -11,6 +11,9 @@ git+https://github.com/changhoonhahn/provabgs.git
 #SN specific dependencies
 sncosmo
 
+# SN baseline specific dependencies
+git+https://github.com/ado8/snmachine
+
```

Comment:
This should be in the readme of the baseline for reconstruction of the baseline. Not in the dataset specific requirements.
```suggestion
```